### PR TITLE
revise computer/calculation setup

### PR DESCRIPTION
**Pull request by ltalirz**

### CODE REVIEW

**Comment by ltalirz** [comment]
I do these PRs in bits to make comparison simpler.
The next step for me would be to
 *  introduce the REST API part in section 2
 * shorten the section on `verdi process list` and move it from section 2 to section 4.4.6 where people need to properly use it for the first time

File: docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst [hunk 1]
```diff
@@ -10,88 +10,89 @@ While we're going to debug these issues 'manually' here, workflows (which you'll
...
-Set up a code
--------------
+Computer setup
+--------------
 
-To run an external code through AiiDA, such as we will be doing with Quantum ESPRESSO's pw.x in this tutorial, we first need to set it up.
-Each code needs to be associated with a computer on which it can run, therefore we first need to configure a computer.
-For the purposes of this tutorial, we are going to run the calculations on the local machine, also referred to as "localhost".
+In a production environment, AiiDA would typically be running on your work station or laptop, while launching calculations
+on some remote high-performance compute resource that you have SSH access to.
```
Comment (odarbelaeze) [inline docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst]: * compute resources?

File: docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst [hunk 2]
```diff
@@ -10,88 +10,89 @@ While we're going to debug these issues 'manually' here, workflows (which you'll
...
+.. code:: console
 
-    The contents of this configuration file will most likely only work for the virtual machine that comes with this tutorial.
-    If you are following this tutorial on your own AiiDA installation, you will have to adapt it yourself.
-    Instructions on how to do this are explained in :ref:`this appendix<appendix_computer_code_setup>`.
+    verdi computer setup --config computer.yml
 
-You can of course also manually copy and paste the content above in a file named ``computer.yml``.
-After you have created or downloaded the file to your local working environment, call the command:
+where ``computer.yml`` is a configuration file in the `YAML format <https://en.wikipedia.org/wiki/YAML#Syntax>`_)  that you can :download:`download here <include/configuration/computer.yml>`. This are its contents:
```
Comment (odarbelaeze) [inline docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst]: * This is its content?

contents is better suited for things like "table of contents": https://dictionary.cambridge.org/grammar/british-grammar/content-or-contents

**Review by odarbelaeze — CHANGES_REQUESTED** [review]
The comments are minimal, this looks very nice already :)

File: docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst [hunk 3]
```diff
@@ -10,88 +10,89 @@ While we're going to debug these issues 'manually' here, workflows (which you'll
...
 
     verdi code setup --config code.yml
 
+.. warning::
+
+    The configuration should work for the virtual machine that comes with this tutorial.
+    If you are following this tutorial in a different environment, you will need to install Quantum ESPRESSO and adapt the configuration to your needs, 
+    as explained in :ref:`this appendix<appendix_computer_code_setup>`.
+
 Similar to the computers, you can list all the configured codes with:
```
Comment (sphuber) [inline docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst:86]: You removed the `verdi computer list` explanation before though

File: docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst [hunk 4]
```diff
@@ -10,88 +10,89 @@ While we're going to debug these issues 'manually' here, workflows (which you'll
...
 
-replacing ``<COMPUTER>`` with the name of the computer you want to configure, for example ``localhost``.
-This will interactively ask for the configuration details, which for a computer using local transport, such as the localhost, is just the one.
 
-.. note::
+Code setup
+----------
 
-    The command we use here is ``verdi computer configure local`` because the ``localhost`` was configured to use ``local`` transport.
-    If you create a computer that requires ``ssh`` transport, the command to configure connection details would be ``verdi computer configure ssh``.
```
Comment (sphuber) [inline docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst:62]: I can see why you would remove this detail, but given that you actually start the section with the fact that "normally" you would configure remote computers with SSH transport, I think this still needs to go in as a note. Unfortunately the syntax `verdi computer configure <TRANSPORT> <COMPUTER>` is not the clearest, due to implementation limitations

**Review by sphuber — CHANGES_REQUESTED** [review]
I would suggest to restore two parts of information that you removed

File: docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst (hunk 3 above)
Comment (ltalirz) [reply docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst:86]: it was still there but in the text. moved it back to .. code::

File: docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst (hunk 4 above)
Comment (ltalirz) [reply docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst:62]: ok, I'm adding a note back

File: docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst (hunk 1 above)
Comment (ltalirz) [reply docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst]: done

File: docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst (hunk 2 above)
Comment (ltalirz) [reply docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst]: agreed, done!

**Comment by ltalirz** [comment]
I'll just go ahead and merge this now, since I've anyhow followed all your suggestions.

**Pull request merged**